libflux/src/semantic/tests.rs

```diff
@@ -2723,3 +2392,391 @@ fn record_with_scoped_labels() {
         ],
     }
 }
+#[test]
+#[ignore]
+fn identify_function() {
+    test_infer! {
+        src: "f = (x) => x",
+        exp: map![
+            "f" => "forall [t0] (x: t0) -> t0",
+        ],
+    }
+}
+#[test]
+#[ignore]
+fn polymorphic_instantiation() {
```

Comment:
booooom. I really like your macros, tests are very expressive in their intent 👍

Reply:
Thank you.